docs/resources/azure_peering_connection.md

```diff
@@ -0,0 +1,136 @@
+---
+page_title: "hcp_azure_peering_connection Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Azure peering connection resource allows you to manage a peering connection between an HVN and a peer Azure VNet.
+---
+
+# hcp_azure_peering_connection (Resource)
+
+-> **Note:** This resource is currently in internal preview only.
+
+The Azure peering connection resource allows you to manage a peering connection between an HVN and a peer Azure VNet.
+
+## Example Usage
+
+```terraform
+resource "hcp_hvn" "hvn" {
+  hvn_id         = "main-hvn"
+  cloud_provider = "azure"
+  region         = "westus2"
+  cidr_block     = "172.25.16.0/20"
+}
+
+resource "hcp_azure_peering_connection" "peer" {
+  hvn_link                 = hcp_hvn.hvn.self_link
+  peering_id               = "dev"
+  peer_vnet_name           = azurerm_virtual_network.vnet.name
+  peer_subscription_id     = "50dc4fb8-db16-4c07-8c35-f31467122911"
+  peer_tenant_id           = "09cdb642-fb81-4425-ba72-b882790ecb06"
+  peer_resource_group_name = azurerm_resource_group.rg.name
+  peer_vnet_region         = azure_rm_virtual_network.region
+}
+
+resource "hcp_peering_connection_activation" "activation" {
```

Comment:
Does this succeed when customer accepts the peering connection? If these two are accurate could you mention this in the docs?

Reply:
Yes! Good reminder - I still need to update the docs with the new direction we took.